### PR TITLE
Fix duplicate fields, add eslint to configs, add travis.

### PR DESCRIPTION
**Pull request by ryan-roemer**
**Status**: Hooking up now to check on Travis. Still have lots of eslint fixes for our configs and rules before ready for review.

Fixes: #54 

Supersedes: #55 #53 

/cc @baer

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name"            : "eslint-config-defaults",
```
Comment (ryan-roemer) [inline package.json]: **Note**: `npm --save` formats `package.json`, so we should just let it... (keeping the spaces removed)

**Comment by ryan-roemer** [comment]
UPDATE: Read for real review and passes Travis!

/cc @baer @JanNorden @agirorn

File: rules/eslint/style/walmart.js [hunk 2]
```diff
@@ -112,8 +112,6 @@ module.exports = {
     "sort-vars": 0,
     // require a space after certain keywords
     "space-after-keywords": [2, "always"],
-    // require a space before certain keywords
-    "space-before-keywords": 0,
```
Comment (baer) [inline rules/eslint/style/walmart.js]: Why is this removed?

File: .eslintrc [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+extends:
+  # We're manually approximating `configurations/walmart/es5-node` with these
```
Comment (baer) [inline .eslintrc]: Why doesn't this just extend `configurations/walmart/es5-node`?

File: rules/eslint/style/walmart.js (hunk 2 above)
Comment (ryan-roemer) [inline rules/eslint/style/walmart.js]: Because it's a **duplicate field that failed strict**.

File: .eslintrc (hunk 3 above)
Comment (ryan-roemer) [inline .eslintrc]: Because it dosn't work. Hence `since `defaults/` prefix assumes npm-installed`
